### PR TITLE
Updated Perf Test readme

### PR DESCRIPTION
**Pull request by manashgoswami**
**Description**: Updated readme to correct version

### CODE REVIEW

File: onnxruntime/test/perftest/README.md [hunk 1]
```diff
@@ -1,25 +1,63 @@
...
         --test_data_set_0
             --input0.pb
         --test_data_set_2
 	        --input0.pb
         --model.onnx
-    The path of model.onnx needs to be provided as <model_path> argument.
+    
+The path of model.onnx needs to be provided as `<model_path>` argument.
+
+__Sample output__ from the tool will look like something this:
```
Comment (askhade) [inline onnxruntime/test/perftest/README.md]: will look something like this: ?

File: onnxruntime/test/perftest/README.md [hunk 2]
```diff
@@ -1,25 +1,63 @@
 # ONNXRuntime Performance Test
 
-onnxruntime_perf_test [options...] model_path result_file
+This tool provides the time for the ONNX Runtime with the specific execution provider to run a inference for a given model using the given test date. This tool can provide a reliable measurement for the inference latency usign ONNX Runtime on the HW platform. The options to use with the tool are listed below:
```
Comment (askhade) [inline onnxruntime/test/perftest/README.md]: test data

Comment (askhade) [reply onnxruntime/test/perftest/README.md]: instead of saying time may be performance data?

File: onnxruntime/test/perftest/README.md (hunk 1 above)
Comment (manashgoswami) [reply onnxruntime/test/perftest/README.md]: fixed.

File: onnxruntime/test/perftest/README.md (hunk 2 above)
Comment (manashgoswami) [reply onnxruntime/test/perftest/README.md]: __sample input__ test data?

Comment (askhade) [reply onnxruntime/test/perftest/README.md]: test data instead of test date

Comment (manashgoswami) [reply onnxruntime/test/perftest/README.md]: updated

File: onnxruntime/test/perftest/README.md [hunk 3]
```diff
@@ -1,25 +1,63 @@
...
 
 Model path and input data dependency:
-    Performance test uses the same input structure as onnx_test_runner. It requrires the directory trees as below:
+    Performance test uses the same input structure as *onnx_test_runner* tool. It requrires the directory trees as below:
 
     --ModelName
         --test_data_set_0
             --input0.pb
         --test_data_set_2
 	        --input0.pb
```
Comment (tianleiwu) [inline onnxruntime/test/perftest/README.md]: aligment

File: onnxruntime/test/perftest/README.md [hunk 4]
```diff
@@ -1,25 +1,63 @@
 # ONNXRuntime Performance Test
 
-onnxruntime_perf_test [options...] model_path result_file
+This tool provides the performance results using the ONNX Runtime with the specific execution provider to run the inference for a given model using the sample input test data. This tool can provide a reliable measurement for the inference latency usign ONNX Runtime on the HW platform. The options to use with the tool are listed below:
```
Comment (tianleiwu) [inline onnxruntime/test/perftest/README.md]: HW is not How about change the HW platform to current platform.

File: onnxruntime/test/perftest/README.md [hunk 5]
```diff
@@ -1,25 +1,63 @@
...
-        -e [cpu|cuda|mkldnn|tensorrt|ngraph|nuphar]: Specifies the provider 'cpu','cuda','mkldnn','tensorrt','ngraph' or 'nuphar'. Default:'cpu'.
-        -r [repeated_times]: Specifies the repeated times if running in 'times' test mode.Default:1000.
-        -t [seconds_to_run]: Specifies the seconds to run for 'duration' mode. Default:600.
-        -p [profile_file]: Specifies the profile name to enable profiling and dump the profile data to the file.
-        -s: Show statistics result, like P75, P90.
-        -v: Show verbose information.
-        -x: Use parallel executor, default (without -x): sequential executor.
-        -h: help
+
+	-A: Disable memory arena
```
Comment (tianleiwu) [inline onnxruntime/test/perftest/README.md]: Append . at the end. Same for -M.

File: onnxruntime/test/perftest/README.md (hunk 4 above)
Comment (manashgoswami) [reply onnxruntime/test/perftest/README.md]: or ... on the __device__?

**Pull request merged**